File: src/main/java/net/atos/zac/productaanvraag/model/InboxProductaanvraagListParameters.java
/*
 * SPDX-FileCopyrightText: 2022 Atos
 * SPDX-License-Identifier: EUPL-1.2+
 */

package net.atos.zac.productaanvraag.model;

import nl.info.zac.search.model.DatumRange;
import nl.info.zac.shared.model.ListParameters;


public class InboxProductaanvraagListParameters extends ListParameters {

    private DatumRange ontvangstdatum;

    private String type;

    private String initiatorID;


    public InboxProductaanvraagListParameters() {
        super();
    }

    public DatumRange getOntvangstdatum() {
        return ontvangstdatum;
    }

    public void setOntvangstdatum(final DatumRange ontvangstdatum) {
        this.ontvangstdatum = ontvangstdatum;
    }

    public String getType() {
        return type;
    }

    public void setType(final String type) {
        this.type = type;
    }

    public String getInitiatorID() {
        return initiatorID;
    }

    public void setInitiatorID(final String initiatorID) {
        this.initiatorID = initiatorID;
    }
}

File: src/main/kotlin/nl/info/zac/productaanvraag/model/InboxProductaanvraagListParameters.kt
/*
 * SPDX-FileCopyrightText: 2022 Atos, 2026 INFO.nl
 * SPDX-License-Identifier: EUPL-1.2+
 */
package nl.info.zac.productaanvraag.model

import nl.info.zac.search.model.DatumRange
import nl.info.zac.shared.model.ListParameters

class InboxProductaanvraagListParameters : ListParameters() {
    var ontvangstdatum: DatumRange? = null
    var type: String? = null
    var initiatorID: String? = null

    override fun toString() =
        "InboxProductaanvraagListParameters{ontvangstdatum=$ontvangstdatum, type=$type, " +
            "initiatorID=$initiatorID, sorting=$sorting, paging=$paging}"
}
